cmd/packaging/packaging.go

```diff
@@ -67,16 +67,25 @@ type packagingTask struct {
 	packagingFunction              func(tmpPath, applicationName, packageName, executableName, version, release string) (string, error) // Function that actually packages the app. Needs to check for OS specific tools etc. . Returns the path of the packaged file
 	skipAssertInitialized          bool                                                                                                 // Set to true when a task doesn't need to be initialized.
 	requiredTools                  map[string][]string                                                                                  // Map of list of tools required to package per OS
+	toolsInstallationInstruction   map[string]map[string]string
```

Comment:
I would prefer that we combine this with requiredTools so that every requiredTool automatically must fill a requirement descirption. Perhaps do this after the merge of #167 though?

So, if we have a `requiredTools map[string][]toolInfo` and a `struct toolInfo{cmd string, installationInstructions string}`, we combine the two properly and cannot have any missing instructions or instructions for tools that we don't even use.

Reply:
Hm some tools don't need a description, because they are standard tools on the OS (I mean preinstalled) so maybe they shouldn't even be in the list. Now that I'm thinking of it the linux tools are mostly GNU tools so they may even be not installed on a linux system.
But merging these sounds good.

Reply:
That's okay, then we can leave the description empty or simply add "Please install xyz" without further specific instructions.

Reply:
Done